### ISSUE TITLE
1.5后<>不可使用

### ISSUE THREAD

**Issue by xiangyi-chen** [nahsra/antisamy#60]
The front desk carries a parameter through the url, the value of the parameter is <>, through Policy policy = Policy.getInstance("/antisamy-slashdot-1.4.4.xml");
final CleanResults cr = antiSamy.scan(value, policy);
String str = cr.getCleanHTML(); 
The str obtained is escaped &lt;&gt;, there is no such problem before 1.5, why not after 1.5

**Comment by spassarop** [comment]
Hi @xiangyi-chen. Before someone takes a look at this, could you provide an input and output example?

For me at least, it's difficult to understand your current input format.

**Comment by xiangyi-chen** [comment]
> 嗨@ xiangyi-chen。在有人对此进行介绍之前，您能提供一个输入和输出示例吗？
> 
> 至少对我来说，很难理解您当前的输入格式。

hi，@Sebastián Passaro。
E.g:The input format is: hello>,<hello
The output format becomes: hello&gt;,&lt;hello
><is escaped into &gt;&lt;
So I want to know what configuration can allow these two special symbols
And using <regexp name="anything" value=".*"/>

**Comment by xiangyi-chen** [comment]
hi，@Sebastián Passaro。
E.g:The input format is: hello&gt;,<hello
The output format becomes: hello&amp;gt;,&amp;lt;hello
&gt;<is escaped into &amp;gt;&amp;lt;
So I want to know what configuration can allow these two special symbols
And using <regexp name="anything" value=".*"/&gt;





Best Regards
Xiangyi.chen
Hangzhou Developer
T: +86 155 0466 1239
Email:&nbsp;chenxiangyi@qq.com


&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "nahsra/antisamy"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年12月15日(星期二) 上午6:59
收件人:&nbsp;"nahsra/antisamy"<antisamy@noreply.github.com&gt;;
抄送:&nbsp;"chenxiangyi@qq.com"<1163211043@qq.com&gt;;"Mention"<mention@noreply.github.com&gt;;
主题:&nbsp;Re: [nahsra/antisamy] 1.5后<&gt;不可使用 (#60)





 
Hi @xiangyi-chen. Before someone takes a look at this, could you provide an input and output example?
 
For me at least, it's difficult to understand your current input format.

**Comment by spassarop** [comment]
@xiangyi-chen, I'm sorry but it seems I didn't catch again which the input is. I've tired on the most recent and 1.4.4 versions and the output is the same for what I understood the input was. 

This is the code I ran:
```java
String input = "hello>,<hello";
CleanResults cr = as.scan(input, policy, AntiSamy.DOM);
System.out.println(cr.getCleanHTML());
```
And my output on the versions I tried is `hello&gt;,`. I used the slashdot policy on 1.4.4 but It changes nothing.
I suggest to copy and paste the code you're using as I did, so I can reproduce this issue the same way as you. Also because on your previous messages the characters are sometimes encoded and sometimes not, so it's inconsistent and I don't know what to use as input.
